src/pykiso/test_setup/dynamic_loader.py

```diff
@@ -229,6 +229,11 @@ def get_instance(self, name: str) -> Union[AuxiliaryCommon, Connector]:
         log.internal_debug(f"instantiated {name}")
         return inst
 
+    def delete_all_instances(self):
+        for instance in self.instances.values():
+            if hasattr(instance, "__del__") and callable(getattr(instance, "__del__")):
+                instance.__del__()
```

Comment:
```suggestion
                del instance
```

Reply:
This never calls the custom __del__ methods in this case